assignments/array-methods.js

```diff
@@ -56,28 +56,45 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function (object){
+    let name = object.first_name + " " + object.last_name;
```

Comment:
Perfect. You can also look into template strings https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals